gap-analysis/index.md

```diff
@@ -328,8 +328,7 @@ The x-coordinate of the point B' could be encoded as:
 where the name “index” is used here to select a component from a tuple, in this case, the x-coordinate from a point.
 
 
-Various proposals have been discussed for the syntax to be supported by the intent attribute, with varying trade-offs in markup convenience. One proposal is to include default intent values for existing MathML presentation forms and default names for math operators to reduce the amount of markup needed to specify the intent of an expression. For example
-\\[ \sqrt{ \frac{180}{5} } = 6 \\] would not require the explicit use of `intent` but would be equivalent to a version that included `intent="square-root"` on `msqrt`, etc. While the full encoding of the content markup for an expression is often complex, and often not included with the presentation, the inclusion of the intent markup for an expression is potentially much simpler to generate and easier to consume, as part of the presentation markup.
+Various proposals have been discussed for the syntax to be supported by the intent attribute, with varying trade-offs in markup convenience. One proposal is to include default intent values for existing MathML presentation forms and default names for math operators to reduce the amount of markup needed to specify the intent of an expression. For example $\frac{n!}{(n-1)!)} = n$ would not require the explicit use of `intent` but would be equivalent to a version that included `intent="factor($1)"` on the `mrow`s in the fraction, etc. While the full encoding of the content markup for an expression is often complex, and often not included with the presentation, the inclusion of the intent markup for an expression is potentially much simpler to generate and easier to consume, as part of the presentation markup.
```

Comment:
ok but I think `intent=factorial($1)`  as factor() left me scratching my head wondering what the implied factorisation was.